### PR TITLE
Hotfix/histogram duplicates or absent

### PR DESCRIPTION
**Pull request by simonamaggio**
- when user selected `top_k_features` to display all feature distributions, some of them were not shown as they have 0 importance for the decision tree, but the user needs to be able to look at the distribution of all features in a node
- when using pipeline, some features hist were duplicated (multiple hist for the one hot encoded features)

### CODE REVIEW

File: mealy/error_visualizer.py [hunk 1]
```diff
@@ -221,8 +221,8 @@ def plot_feature_distributions_on_leaves(self, leaf_selector='all_errors',
             min_values, max_values = x.min(axis=0), x.max(axis=0)
             feature_names = self.mpp_feature_names
         else:
-            ranked_feature_ids = [self.pipeline_preprocessor.inverse_transform_feature_id(idx) for idx in
-                                  ranked_feature_ids]
+            ranked_feature_ids = list(set([self.pipeline_preprocessor.inverse_transform_feature_id(idx) for idx in
+                                  ranked_feature_ids]))
```
Comment (AgatheG) [inline mealy/error_visualizer.py:225]:
```suggestion
            ranked_feature_ids = list(set(self.pipeline_preprocessor.inverse_transform_feature_id(idx) for idx in
                                  ranked_feature_ids))
```

**Review by AgatheG — CHANGES_REQUESTED** [review]
Are the duplicates really duplicates? Is it not that for one hot encoded categorical features, you get a distribution for each category?

**Comment by simonamaggio** [comment]
> Are the duplicates really duplicates? Is it not that for one hot encoded categorical features, you get a distribution for each category?

yes, exactly

**Comment by simonamaggio** [comment]
@AgatheG are you ok with those changes?

**Review by AgatheG — APPROVED** [review]

**Pull request merged**